common_functions.sh

```diff
@@ -97,6 +97,17 @@ function set_arch_os() {
 				current_arch="x86_64"
 				#  windowsservercore-1809 windowsservercore-1803 are not included as Adopt can't build them
 				oses="windowsservercore-ltsc2016"
+				# this variable will only be set when running under GitHub Actions
+				if [ -n "${BUILD_OS}" ]; then
+					case ${BUILD_OS} in
+					windows-2019)
+						oses="windowsservercore-1809 nanoserver-1809 windowsservercore-1909 nanoserver-1909 windowsservercore-ltsc2019"
+						;;
+					windows-2016)
+						oses="windowsservercore-ltsc2016"
+						;;
+					esac
```

Comment:
Do we need this for Ubuntu as well ?

Reply:
No, there is no need for ubuntu. The problem with Windows is that you can only build images for the Windows version you are currently running on (this can be overcome, but apparently not on GitHub Actions). With ubuntu, you can build any Linux docker image you want, so I didn't set the oses that would be built for ubuntu.